docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -238,15 +238,19 @@ as a distribution method, but you can use it directly to install the collection
 Trying collection locally
 -------------------------
 
-You can try your collection locally by installing it from the tarball.
+You can try your collection locally by installing it from the tarball. The following will enable an adjacent playbook to
+access the collection:
 
 .. code-block:: bash
 
-   ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections/ansible_collections
+   ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections
```

Comment:
@acozine @chouseknecht **If** this PR as a whole gets deferred to 2.10, should the above two changes be pushed into 2.9.x separately?  It seems like it fixes an independent documentation bug.

Reply:
I am pretty positive that the change to give a default install path will not be accepted for 2.9, potentially not even for 2.10.